websecurityscanner/noxfile.py

```diff
@@ -23,40 +23,36 @@
 LOCAL_DEPS = (os.path.join("..", "api_core"), os.path.join("..", "core"))
 
 @nox.session(python="3.7")
-def blacken(session):
-    """Run black.
+def lint(session):
+    """Run linters.
 
-    Format code to uniform standard.
+    Returns a failure if the linters find linting errors or sufficiently
+    serious code quality issues.
     """
-    session.install("black")
+    session.install("flake8", "black", *LOCAL_DEPS)
     session.run(
         "black",
+        "--check",
         "google",
         "tests",
         "docs",
-        "--exclude",
-        ".*/proto/.*|.*/gapic/.*|.*/.*_pb2.py",
     )
+    session.run("flake8", "google", "tests")
 
 
-@nox.session(python="3.7")
-def lint(session):
-    """Run linters.
+@nox.session(python="3.6")
+def blacken(session):
+    """Run black.
```

Comment:
@crwilcox Again, why would we run lint first?

Reply:
if you run blacken first, it will make linting a noop as it is going to modify the source. A possible modification would be to remove blacken as a target that runs on a no-argument call to `nox`

Reply:
`lint` might still complain about stuff that `black` doesn't fix (unused imports / arguments, for instance?)  But failing CI when `black` *could* fix it seems pointless.